stock_quant_serial_partner_location/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,4 @@
+This module remove the check on serial number quants when the location is type customer or vendor.
+It is usefull to specific situation, when you sell products to one company and then the same one provide
+dropship service. When you try to send to the final customer the same serial number the error will pop up.
+This module block the raise and allow you to send it on that situation.
```

Comment:
Corrected text:

This module removes the check on serial number quants when the location is type customer or vendor.
It is useful to specific situation, when you sell products to one company and then the same one provides a
dropship service. When you try to send to the final customer the same serial number the error will pop up.
This module blocks the raise and allows you to send it in that situation.

Reply:
Done, Thanks